src/corelibs/U2Algorithm/src/misc/RollingMatrix.h

```diff
@@ -97,6 +98,10 @@ class U2ALGORITHM_EXPORT RollingMatrix {
     }
 
 protected:
+    static qint64 getMatrixSizeInBytes(int sizeX, int sizeY) {
```

Comment:
Keep it public as before: other algorithms that create a lot of matrices and have memory limits may call it